### ISSUE TITLE
focus() on bal-input prevents updating formcontrol value - only in cypress

### ISSUE THREAD

**Issue by tklueber** [baloise-incubator/design-system#303]
I would like to describe the following issue here, which may have to do with the design system, because it only occurs with the change from **input** to **bal-input.**

The following behaviour **only occurs in cypress webtests** and I have not yet been able to manually reproduce it directly in an application:

**Testcase Angular App:**
1. fill bal-input with a value -> formcontrol value is set reactive. 
2. delete value from the field again (cy.get(fieldselector).clear())
3. remove focus so that the vaildation message appears (cy.get(fieldselector).focus().blur() )

The result is that even though the field has been cleared and is no longer a value in the DOM, the formcontrol itself still has the original value.

However, this occurs if focus() was previously called on the field. Perhaps there is a solution for this or the reason can be determined;

tl;dr: Only occurs in cypress; formcontrol value only remains if the focus() was previously called in webtest, 

maybe this code and screenshot help to understand:
```
    cy.get(basicsPage.emailComponent).type('some.one@example.com');
    cy.get(basicsPage.emailComponent).focus();
    cy.get(basicsPage.emailComponent).blur();
    cy.get(basicsPage.emailComponent).clear();
```
![image](https://user-images.githubusercontent.com/292344/183053812-365642a3-2037-4d72-a142-3b1c881d95a1.png)

**Comment by nobilo** [comment]
It seems like this happens only when you combine focus and then blur. When we combine all tree stuff:
```
  cy.get(basicsPage.emailComponent).type('some.one@example.com');
  cy.get(basicsPage.emailComponent).focus();
  cy.get(basicsPage.emailComponent).blur();
```
then even without clear the input field gets cleared, however having in mind you already interacting with element by typing you do not need focus so with just remove:
`  cy.get(basicsPage.emailComponent).focus();
`
and then it works just fine.
